docs/GettingStarted.md

```diff
@@ -2032,7 +2032,6 @@ end
 | `tags`                                                  | `DD_TAGS`                      | `nil`                                                             | Custom tags in value pairs separated by `,` (e.g. `layer:api,team:intake`) These tags are set on all traces. See [Environment and tags](#environment-and-tags) for more details.                                                          |
 | `time_now_provider`                                     |                                | `->{ Time.now }`                                                  | Changes how time is retrieved. See [Setting the time provider](#Setting the time provider) for more details.                                                                                                                              |
 | `version`                                               | `DD_VERSION`                   | `nil`                                                             | Your application version (e.g. `2.5`, `202003181415`, `1.3-alpha`, etc.) This value is set as a tag on all traces.                                                                                                                        |
-| `telemetry.enabled`                                     | `DD_INSTRUMENTATION_TELEMETRY_ENABLED` | `true`                                                             | Allows you to opt-out of sending telemetry data to Datadog. See [Telemetry collection](https://docs.datadoghq.com/tracing/configure_data_security/#telemetry-collection) for more details.                                                                                                                                                                                          |
```

Comment:
Since this remains an option that users can configure, should we keep the documentation here, but mention that it's enabled by default? (But we expect this will change in a future release?)

Reply:
Updated!